### ISSUE TITLE
delay

### ISSUE THREAD

**Issue by walidashi** [Androz2091/discord-player#80]
hey when I -play (song) I get this error but after a couple of seconds it works

```

(node:10476) UnhandledPromiseRejectionWarning: Error: Not playing
    at C:\Users\MH\Documents\Coding\music bot\node_modules\discord-player\src\Player.js:501:39
    at new Promise (<anonymous>)
    at Player.addToQueue (C:\Users\MH\Documents\Coding\music bot\node_modules\discord-player\src\Player.js:498:16)
    at Client.<anonymous> (C:\Users\MH\Documents\Coding\music bot\index.js:51:52)
    at Client.emit (events.js:326:22)
    at MessageCreateAction.handle (C:\Users\MH\Documents\Coding\music bot\node_modules\discord.js\src\client\actions\MessageCreate.js:31:14)
    at Object.module.exports [as MESSAGE_CREATE] (C:\Users\MH\Documents\Coding\music bot\node_modules\discord.js\src\client\websocket\handlers\MESSAGE_CREATE.js:4:32)
    at WebSocketManager.handlePacket (C:\Users\MH\Documents\Coding\music bot\node_modules\discord.js\src\client\websocket\WebSocketManager.js:384:31)
    at WebSocketShard.onPacket (C:\Users\MH\Documents\Coding\music bot\node_modules\discord.js\src\client\websocket\WebSocketShard.js:444:22)
    at WebSocketShard.onMessage (C:\Users\MH\Documents\Coding\music bot\node_modules\discord.js\src\client\websocket\WebSocketShard.js:301:10)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:10476) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:10476) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
/*****************************************************************************************************
 * validateURL will be renamed to validateID to match with getPlaylistID in the next release of ytpl *
 *                   set `ytpl.do_warn_deprecate = false` to disable this message                    *
 *****************************************************************************************************/```

**Comment by walidashi** [comment]
this is what written in line 51
```const result = await client.player.addToQueue(message.member.voice.channel, args[0]);```

**Comment by Androz2091** [comment]
This is fixed in v3.0.0-beta.3
